### DISCUSSION TITLE
Merge AMCD-formatter & Smashforge

### DISCUSSION THREAD

**Discussion by Bibidimah21** [ScanMountGoat/ssbh_editor#89] [Ideas]
Get merge with ACMD-formatter with Smashforge and Successor to Smashforge & BrawlCrate to be a better glory.

**Comment by ScanMountGoat** [comment]
SSBH Editor is focused on viewing and editing model files rather than moveset edits. Keeping the application focused helps to make the code easier to maintain and avoids complicating the interface with too many features. There wouldn't be much benefit to merging with Smash Forge and Brawl Box beyond having fewer desktop icons.

I may consider adding hitbox/hurtbox rendering to SSBH Editor at some point, but there aren't any plans to work on that at this time.

**Reply by Bibidimah21** [reply]
.NRO AMCD Hitboxes moveset be a adding update SSBH edit to be easy?

**Reply by ScanMountGoat** [reply]
Drawing the hitbox shapes is easy. The hard part is getting the hitboxes into a format that SSBH Editor can use. This would be something the rest of the community would have to figure out, since I don't work with ACMD stuff myself.

**Reply by ScanMountGoat** [reply]
See https://github.com/ScanMountGoat/ssbh_editor/issues/146#issuecomment-4320694025 for a WIP external tool that can view hitboxes. https://github.com/Common-Leap/Hitbox-editor
